### DISCUSSION TITLE
Dúvida sobre publicação dos projetos para testar

### DISCUSSION THREAD

**Discussion by dalmiroe** [balta-io/forum#451] [Tira Dúvidas] [answered]
Bom dia. Estou fazendo os cursos mas ainda não vi nada sobre a parte de publicação. Para os projetos blazor, a publicação é somente no azure? Exemplo, tenho uma conta na locaweb, vou conseguir publicar lá? Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @dalmiroe, como vai?

Nós temos um curso só sobre publicação, ele está na carreira 04 (DevOps). É importante que faça todos os cursos na ordem e não pule nenhum.

A publicação de APis e Apps Blazor é a mesma coisa... na verdade qualquer publicação de aplicações .NET será sempre igual.

**Comment by dalmiroe** [comment]
Perfeito, obrigado
